### PR TITLE
button modified

### PR DESCRIPTION
**Pull request by Priyansh61**
The toggle button has been modified to the below button

### Proof:

https://user-images.githubusercontent.com/96513964/194313267-b46d1701-1728-434b-8e37-f20d053d0e4c.mp4

### CODE REVIEW

**Comment by Priyansh61** [comment]
Hey @immattdavison  I have added a new branch to keep the commits clean and synced with the master branch.

**Comment by immattdavison** [comment]
> Hey @immattdavison I have added a new branch to keep the commits clean and synced with the master branch.

Hey @Priyansh61 I see you have tested this as a webpage, could you show it as a browser extension so that we can view the functionality

**Comment by Priyansh61** [comment]
https://user-images.githubusercontent.com/96513964/194342547-c6e0ca63-accd-42fa-98af-eaef356f1383.mp4

Is this al right @immattdavison ?

**Comment by immattdavison** [comment]
I like that!

**Comment by Priyansh61** [comment]
Thanks a lot @immattdavison  for guiding me.

Do i need to make any further changes?

File: popup.html [hunk 1]
```diff
@@ -1,30 +1,34 @@
...
-    <link rel="icon" href="assets/icon.png" />
-    <link rel="stylesheet" href="style.css" />
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-    <title>No More Domains</title>
-  </head>
+  <link rel="icon" href="assets/icon.png" />
```
Comment (immattdavison) [inline popup.html:9]: this is not necessary - please remove

File: popup.html [hunk 2]
```diff
@@ -1,30 +1,34 @@
...
-        <input type="checkbox" id="toggleButton" checked>
-        <span class="slider round"></span>
-      </label>
-    </div>
-    <script src="blockpage.js" type="text/javascript"></script>
-    <script src="popup.js" type="text/javascript"></script>
-  </body>
-</html>
+  <title>No More Domains</title>
+  <style>
```
Comment (immattdavison) [inline popup.html:13]: This style tag serves no purpose

**Review by immattdavison — CHANGES_REQUESTED** [review]

File: popup.html (hunk 2 above)
Comment (Priyansh61) [reply popup.html]: removed all the unecessary elements

**Comment by Priyansh61** [comment]
@immattdavison I have made all the requested changes, PTAL!

**Comment by Priyansh61** [comment]
@immattdavison Thank you so much... Learnt a lot from you :smile: 

Do let me know if  there is any other issue for me to work on ?

**Comment by immattdavison** [comment]
I'd say just take a look through the current issues, I know #65 needs some help

**Pull request merged**